examples/authentication/mtls/mtls-vault.yml

```
@@ -0,0 +1,16 @@
+quarkus:
+  file:
+    vault:
+      provider:
+        pim:
+          path: config/pimvault.p12
+          secret: ${vault.storepassword} # This will be provided as a property
+  http:
+    ssl:
+      certificate:
+        credentials-provider: quarkus.file.vault.provider.pim
+        key-store-file: config/server-keystore.jks
+        key-store-alias: pim
```

```suggestion
        key-store-key-alias: pim
```

The other one doesn't exist (or couldn't find it at least at Quarkus documentation)

---

if it is not the right key, then I should remove it completely from the example as the defaults worked as expected